protocol-definitions/MC.yaml

```diff
@@ -981,3 +981,61 @@ methods:
           since: 2.8
           doc: |
             Member configuration file content (XML or YAML string) of the member or null if the server was not started using a configuration file.
+  - id: 39
+    name: setDiagnosticsConfig
+    since: 2.9
+    doc: |
+      Sets the diagnostics configuration on a running cluster.
+    request:
+      retryable: false
+      partitionIdentifier: -1
+      params:
+        - name: enabled
+          type: boolean
+          nullable: false
+          since: 2.9
+          doc: |
+            Indicates whether diagnostics service is enabled or not
+        - name: outputType
+          type: String
+          nullable: false
+          since: 2.9
+          doc: |
+            The output type for the diagnostics
+        - name: includeEpochTime
+          type: boolean
+          nullable: false
+          since: 2.9
+          doc: |
+            Indicates if the epoch time should be included in the 'top' section
+        - name: maxRolledFileSizeInMB
+          type: int
+          nullable: false
+          since: 2.9
+          doc: |
+            The maximum size in MB for a single file
+        - name: maxRolledFileCount
+          type: int
+          nullable: false
+          since: 2.9
+          doc: |
+            The maximum number of rolling files to keep on disk
+        - name: logDirectory
+          type: String
+          nullable: false
+          since: 2.9
+          doc: |
+            The output directory of the diagnostics log files
```

Comment:
Can you indicate if this is a file path or not and if this is relative path or absolute path? Can you also may be put an example of such a path.

Reply:
fixed

Reply:
can you also write if it is a relative or absolute path or may be both?

Reply:
fixed